mods/cnc/chrome/lobby-mappreview.yaml

```diff
@@ -36,6 +30,14 @@ Container@LOBBY_MAP_PREVIEW:
 					Height: 25
 					Font: Tiny
 					Align: Center
+				LabelWithTooltip@MAP_TITLE:		# MAP_TITLE is displayed above MAP_TYPE, MAP_AUTHOR, but to be in front of them for tooltip it was moved here below
```

Comment:
The tooltip is attached to the `TooltipContainer`, so the position of this label shouldn't matter at all?